lib/mongo/collection.rb

```diff
@@ -184,11 +190,11 @@ def create
       if (options[:collation] || options[Operation::COLLATION]) && !server.features.collation_enabled?
         raise Error::UnsupportedCollation.new
       end
-      Operation::Commands::Create.new({
-                                        selector: operation,
-                                        db_name: database.name,
-                                        write_concern: write_concern
-                                      }).execute(server)
+      Operation::Commands::Create.new(
```

Comment:
That's a good question - one that I asked myself as well.
The spec says 'The server reports the operation time in the response to CRUD commands.'
The ``with_session`` method extracts the operationTime from the operation response and given that the spec says the value is reported in CRUD commands, I've only wrapped CRUD commands in the ``with_session`` method. I'll keep my eye on the spec though, and test other commands, to see if operationTime is reported in more than just ones of the CRUD type.